clang/test/3C/canwrite_constraints.h

```diff
@@ -71,6 +71,13 @@ void unwritable_func_with_itype_and_bounds(int *p
 // expected-warning@+1 {{Declaration in non-writable file}}
 _Itype_for_any(T) void my_generic_function(void *p : itype(_Ptr<T>));
 
+// expected-warning@+1 {{Declaration in non-writable file}}
+_Itype_for_any(T) void* my_generic_return(void) : itype(_Ptr<T>);
```

Comment:
Code formatting nit.  (We currently aren't enforcing full `clang-format` of `clang/test/3C`, but I thought it might be a good opportunity to remind you of our standard format.)
```suggestion
_Itype_for_any(T) void *my_generic_return(void) : itype(_Ptr<T>);
```